### DISCUSSION TITLE
Improve and add new features/post-processing effects to the Camera 3D.

### DISCUSSION THREAD

**Discussion by Dynacato** [Redot-Engine/redot-proposals#60] [Ideas]
Options like camera aperture/auto exposure, dynamic range, focus, motion blur and lens option on the Camera3D node.
Including more tonal mappings and a better color correction system.
I don't know which other features I am missing here but as I seen in other engines, it would be a great addition.

**Comment by Arctis-Fireblight** [comment]
I think we have a contributor working on motion blur, so should hopefully get a PR for that coming soon.

Regarding some of the other effects you mentioned, would you be willing to elaborate a little more on how you'd like to see them implemented so we could potentially turn this into a design spec / formal proposal that can be implemented?
